Pokedex_ios/View/Controller/AdvanceFilterPopup/AdvancedFilterPopupController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  AdvancedFilterPopupController.swift
+//  Pokedex_ios
+//
+//  Created by Alvaro Choque on 5/7/22.
+//
+
+import Foundation
+import UIKit
+
+class AdvancedFilterPopup: PopupViewController {
+    lazy var lineView: UIView = {
+        let line = UIView()
+        line.backgroundColor = .lightGray
+        line.frame = CGRect(x: 0, y: -200, width: 30, height: 350)
+        return line
+    }()
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Filters"
+        label.font = .boldSystemFont(ofSize: 20)
+        return label
+    }()
+    lazy var notesLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Use advanced search to explore Pokémon by type, weakness, height and more!"
+        label.font = .systemFont(ofSize: 16)
+        label.textColor = .darkGray
+        label.numberOfLines = 0
+        return label
+    }()
+    lazy var viewModel = {
+        AdvancedFilterPopupViewModel()
+    }()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initViewModel()
+        setupContent()
+    }
+    func initViewModel() {
+        viewModel.closePopup = { [weak self] in
+            self?.animateDismissView()
+        }
+    }
+    func setupContent() {
+        lazy var resetButton: UIButton = {
```

Comment:
Isn’t it possible to use a nib or a storyboard to create the interface here?

Reply:
just saying because the rest of the component will be added inside this popup, and is a big one, so its better if its supported from a design file